guide/16-geoenrichment-guides/part5_generate_reports.ipynb

```diff
@@ -0,0 +1,928 @@
+{
```

Comment:
<p>"You can create a buffer ring or drive time service area around points of interest. Let's create a report for a street address with non-overlapping disks of radii 1, 3 and 5 Miles respectively using the Business Summary <code>business_summary</code> report. "</p><ul><li>"Reports can also be created for one or more buffered rings, or drive time service areas around the points of interest. Let's create a report for a street address with non-overlapping disks of radii 1, 3 and 5 miles respectively using the Business Summary report template (report="<code>business_summary</code>")."</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>